src/__tests__/Search.test.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { render, screen, fireEvent } from "@testing-library/react";
+import Search from "../components/Search";
+
+describe("Search", () => {
+  it("renders correctly", () => {
+    const { asFragment } = render(<Search />);
+
+    expect(asFragment()).toMatchSnapshot();
+  });
+
+  it("button exists", () => {
+    render(<Search />);
+
+    const searchButton = screen.getByRole("button", { name: /Search/i });
+
+    expect(searchButton).toBeInTheDocument();
+  });
+
+  it("updates input value correctly", () => {
```

Comment:
Could another test be added here to assert that the handler function is called when the form is submitted?
Also, you arent passing in the required function prop to each instance of the component here, you should create a jest.fn() mock this and pass it into each instance